pkg/ebpf/probes/probes.go

```diff
@@ -7,26 +7,28 @@ import (
 	bpf "github.com/aquasecurity/libbpfgo"
 )
 
+// Probes
+
 type Probes interface {
+	// attach
 	Attach(probe Handle) error
+	AttachNetProbes() error
+	// detach
 	Detach(probe Handle) error
+	DetachNetProbes() error
+	// no autoload
+	DisableNetProbesAutoload() error
```

Comment:
It's reasonable to place this function in this package, but I don't think as part of the interface. I think this will occur naturally if we just treat the net probes the same way we treat the regular probes.
And if we do make the net probes part of the standard probes, this logic could possibly fit better in tracee itself (more of a stylistic decision but I do think there is some merit to keeping the interface minimal).